scripts/release.sh

```diff
@@ -498,9 +498,23 @@ function publish_artifacts() {
 
 # Entry point for a release script.
 function main() {
+  parse_flags "$@"
+
+  # Checkout specific branch, if necessary
+  local current_branch="$(git rev-parse --abbrev-ref HEAD)"
```

Comment:
I know it's rare, but `local` swallow the error after `=`
```suggestion
  local current_branch
  current_branch="$(git rev-parse --abbrev-ref HEAD)"
```